vimdoc/module.py

```diff
@@ -112,44 +112,47 @@ def Close(self):
 
     All default sections that have not been overridden will be created.
     """
-    if self.GetCollection(vimdoc.FUNCTION) and 'functions' not in self.sections:
-      functions = Block(vimdoc.SECTION)
-      functions.Local(id='functions', name='Functions')
-      self.Merge(functions)
-    if (self.GetCollection(vimdoc.EXCEPTION)
-        and 'exceptions' not in self.sections):
-      exceptions = Block(vimdoc.SECTION)
-      exceptions.Local(id='exceptions', name='Exceptions')
-      self.Merge(exceptions)
-    if self.GetCollection(vimdoc.COMMAND) and 'commands' not in self.sections:
-      commands = Block(vimdoc.SECTION)
-      commands.Local(id='commands', name='Commands')
-      self.Merge(commands)
-    if self.GetCollection(vimdoc.DICTIONARY) and 'dicts' not in self.sections:
-      dicts = Block(vimdoc.SECTION)
-      dicts.Local(id='dicts', name='Dictionaries')
-      self.Merge(dicts)
+    # ----------------------------------------------------------
+    # Add default sections.
+
+    # type : (id, name)
+    default_sections = {
```

Comment:
I think you need a list of triples or an OrderedDict here to keep section ordering consistent.